Brokerages/GDAX/BestBidAskUpdatedEventArgs.cs

```diff
@@ -0,0 +1,67 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+
+namespace QuantConnect.Brokerages.GDAX
+{
+    /// <summary>
+    /// Event arguments class for the <see cref="OrderBook.BestBidAskUpdated"/> event
+    /// </summary>
+    public sealed class BestBidAskUpdatedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Gets the new best bid price
+        /// </summary>
+        public readonly Symbol Symbol;
```

Comment:
No action required, but curious as to why the preference of field over read-only property here? Typical C# conventions say we should prefer properties. In this specific case I don't see arguments one way or another, but just curious as to what inspired the decision

Reply:
No reason at all to use fields, should be get properties, I'll fix it now